includes/Emails/ContactSeller.php

```diff
@@ -52,56 +62,68 @@ public function get_default_subject() {
      * @return string
      */
     public function get_default_heading() {
-            return __( '{customer_name} - Sent a message from {site_name}', 'dokan-lite' );
+        return __( '{customer_name} - Sent a message from {site_name}', 'dokan-lite' );
     }
 
     /**
      * Trigger the this email.
      */
     public function trigger( $seller_email, $contact_name, $contact_email, $contact_message ) {
-		if ( ! $this->is_enabled() || ! $this->get_recipient() ) {
-			return;
-		}
-
-            $seller = get_user_by( 'email', $seller_email );
-
-            $this->find['seller_name']    = '{seller_name}';
-            $this->find['customer_name']  = '{customer_name}';
-            $this->find['customer_email'] = '{customer_email}';
-            $this->find['message']        = '{message}';
-            $this->find['site_name']      = '{site_name}';
-            $this->find['site_url']       = '{site_url}';
-
-            $this->replace['seller_name']    = $seller->display_name;
-            $this->replace['customer_name']  = $contact_name;
-            $this->replace['customer_email'] = $contact_email;
-            $this->replace['message']        = $contact_message;
-            $this->replace['site_name']      = $this->get_from_name();
-            $this->replace['site_url']       = site_url();
-
-            $this->setup_locale();
-            $this->send( $seller_email, $this->get_subject(), $this->get_content(), $this->get_headers(), $this->get_attachments() );
-            $this->restore_locale();
+        if ( ! $this->is_enabled() || ! $this->get_recipient() ) {
+            return;
+        }
+
+        $this->reply_to = $contact_email;
+        $seller = get_user_by( 'email', $seller_email );
+
+        $this->find['seller_name']    = '{seller_name}';
+        $this->find['customer_name']  = '{customer_name}';
+        $this->find['customer_email'] = '{customer_email}';
+        $this->find['message']        = '{message}';
+        $this->find['site_name']      = '{site_name}';
+        $this->find['site_url']       = '{site_url}';
+
+        $this->replace['seller_name']    = $seller->display_name;
+        $this->replace['customer_name']  = $contact_name;
+        $this->replace['customer_email'] = $contact_email;
+        $this->replace['message']        = $contact_message;
+        $this->replace['site_name']      = $this->get_from_name();
+        $this->replace['site_url']       = site_url();
+
+        $this->setup_locale();
+        $this->send( $seller_email, $this->get_subject(), $this->get_content(), $this->get_headers(), $this->get_attachments() );
+        $this->restore_locale();
+    }
+
+    /**
+     * Set customer email address for reply
+     *
+     * @since DOKAN_LITE_SINCE
+     *
+     * @return mixed
```

Comment:
This will return must be a `string`, right ??
We should avoid `mixed` wherever possible.